crates/bevy_asset/src/server/mod.rs

```diff
@@ -1109,7 +1187,7 @@ pub enum RecursiveDependencyLoadState {
 }
 
 /// An error that occurs during an [`Asset`] load.
-#[derive(Error, Debug)]
+#[derive(Error, Debug, Clone)]
```

Comment:
PartialEq here too if possible.

Reply:
We unfortunately can't easily add `PartialEq` here because of `AssetLoadError::AssetLoaderError` holding `dyn std::error::Error`. Adding a `PartialEq` trait bound to would create a headache for loaders. If you'd like, I can add a manual implementation of `PartialEq` for `AssetLoadError` that stringifies the error and tests equality on that... but that seems kinda sketchy.